README.md

```diff
@@ -3,26 +3,34 @@ The Azure App Services extension for VS Code lets you quickly browse, create, ma
 
 ## Features
 
-* Browse sites across all of your Azure subscriptions
-* Browse to the Azure Portal for advanced tasks, such as scaling
-* Create new web apps/deployment slots (Linux with Node.js only)
-* Deploy to your web apps/deployment slots
+  * Browse sites across all of your Azure subscriptions
+  * Browse to the Azure Portal for advanced tasks, such as scaling
+  * Create new web apps/deployment slots (Linux with Node.js only)
+  * Deploy to your web apps/deployment slots
+  * View and edit remote files on your web apps (Preview)
```

Comment:
File editing has it's entire section below, so I'm not sure we need it here. I could go either way on this though. Up to you